### DISCUSSION TITLE
Adding github buttons

### DISCUSSION THREAD

**Discussion by stevej2608** [solidjs/solid#1701] [Q&A]
Hi,

I'd like to embed a GitHub stars [button](https://buttons.github.io/) in my first SolidJS project. The accompanying code in [buttons.js](https://github.com/buttons/github-buttons/blob/main/dist/buttons.js) does not play well with SolidJS. On a vanilla html page It creates a shadow DOM and injects the required button markup & a ton of CSS.

Any ideas on how I might get this to work with SolidJS would be very welcome.

Cheers.

**Comment by mdynnl** [comment]
https://playground.solidjs.com/anonymous/e772e15f-0867-4dbd-8bfb-5363fb80b364

**Comment by stevej2608** [comment]
Hay @mdynnl, perfect! Thanks.
